### PR TITLE
Allow multiple pronouns

### PR DESCRIPTION
**Pull request by abompard**
Make `fasPronoun` multi-valued.

Related to https://github.com/fedora-infra/noggin/issues/646.

### CODE REVIEW

File: ui/js/plugins/userfas/userfas.js [hunk 1]
```diff
@@ -28,9 +28,6 @@ define([
                 fields: [{
                     name: 'fastimezone',
                     flags: ['w_if_no_aci']
-                }, {
```
Comment (tiran) [inline ui/js/plugins/userfas/userfas.js:31]: Why did you remove the timezone?

Comment (abompard) [reply ui/js/plugins/userfas/userfas.js:31]: It was a duplicate, there's the same set of lines right above it.

Comment (Conan-Kudo) [reply ui/js/plugins/userfas/userfas.js:31]: Wouldn't it make sense to split that unrelated change out into its own commit?

Comment (abompard) [reply ui/js/plugins/userfas/userfas.js:31]: Sure.
